libraries/chain/webassembly/privileged.cpp

```diff
@@ -134,7 +134,7 @@ namespace eosio { namespace chain { namespace webassembly {
       auto& params = gpo.wasm_configuration;
       uint32_t version = std::min( max_version, uint32_t(0) );
 
-      auto s = fc::raw::pack_size( params );
+      auto s = fc::raw::pack_size( version ) + fc::raw::pack_size( params );
```

Comment:
This will have to be part of a protocol feature change as this is a consensus level change.

Reply:
The protocol feature was added.